ControlRoomApplication/ControlRoomApplicationTest/EntityControllersTests/RadioTelescopeControllerTest.cs

```diff
@@ -1185,6 +1185,39 @@ public void TestAllTelescopeMovements_RunABunchOfCommandsAtTheSameTime_OnlyOneCo
             Assert.AreEqual(1, successes);
             Assert.AreEqual(threadCount - 1, alreadyMovings);
         }
+
+        /// <summary>
+        /// Test method that checks for discrepancy between motor and absolute encoders (to see if motor skips steps/is behind)
+        /// </summary>
+        [TestMethod]
+        public void TestCompareMotorAndAbsoluteEncoders()
```

Comment:
You can break this test up into multiple smaller tests. Smaller tests allow us to test more in case of failures. For example, this test could fail for the minor discrepancy, and the asserts for the major discrepancy and both major discrepancy would never be run. You can follow the naming convention used for other unit tests by separating the method name with a `_` and the specific scenario you are testing.

- [ ] Break into smaller unit tests